_search-plugins/sql/sql/index.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: SQL
+parent: SQL & PPL
+nav_order: 4
+has_children: true
+redirect_from:
+  - /search-plugins/sql/sql
+---
+
+# SQL
+
+## Workbench
+
+The easiest way to get familiar with the SQL plugin is to use **Query Workbench** in OpenSearch Dashboards to test various queries. To learn more, see [Workbench]({{site.url}}{{site.baseurl}}/search-plugins/sql/workbench/).
+
+![OpenSearch Dashboards SQL UI plugin]({{site.url}}{{site.baseurl}}/images/sql.png)
+
+
+## REST API
```

Comment:
We can probably just say "you can use the REST API in the Dev Tools console" and refer them to the API section.